### ISSUE TITLE
.ang phase labeling

### ISSUE THREAD

**Issue by EBRaeker** [EMsoft-org/EMSphInx#10]
I opened .ang files in OIM Analysis that were created by EMSphInx with two phases selected for indexing, however, OIM Analysis was attributing all pixels to one of the two phases. Looking at the documentation on .ang formatting, OIM Analysis is expecting multiphase files to be labeled starting at phase 1 rather than phase 0. EMSphInx labels the multiple phases starting at phase 0 which seems to signal OIM Analysis to treat the file as containing data for a single phase. That is problematic for processing the data further.

The problem seems to be just the phase numbering because using find & replace to relabel the phases starting at phase 1 was successful in having OIM Analysis correctly read in the data as consisting of multiple phases.

**Comment by hakonanes** [comment]
> Looking at the documentation on .ang formatting

Out of curiosity, since I've implemented readers/writers of this format previously, which documentation is this? Is this something else than the OIM manuals? I can't remember having read a specification of the format anywhere.

**Comment by EBRaeker** [comment]
No, I found it within OIM. I have attached the relevant pages of the manual. Phase numbering is near the bottom of the first page.
[OIM ang format.pdf](https://github.com/EMsoft-org/EMSphInx/files/6858912/OIM.ang.format.pdf)

Also, here is the header plus a few lines of data for a multiphase ang file exported from OIM with the columns' data types labeled at the end of the header. 
[OIM ang export.txt](https://github.com/EMsoft-org/EMSphInx/files/6858934/OIM.ang.export.txt)
